### PR TITLE
Use whitespace around operator definitions.

### PR DESCRIPTION
**Pull request by jpsim**
Minor tweaks & refactoring from #82. All credit to @akirahrkw! :clap:

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/ForceCastRule.swift [hunk 1]
```diff
@@ -26,8 +26,7 @@ public struct ForceCastRule: Rule {
         ruleName: "Force Cast Rule",
         ruleDescription: "This rule checks whether you don't do force casts.",
         nonTriggeringExamples: [
-            "NSNumber() as? Int\n",
```
Comment (segiddins) [inline Source/SwiftLintFramework/Rules/ForceCastRule.swift]: why this change?

Comment (jpsim) [inline Source/SwiftLintFramework/Rules/ForceCastRule.swift]: Because of `commentDoesntViolate` in [`verifyRule()`](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFrameworkTests/TestHelpers.swift#L18).

Comment (segiddins) [inline Source/SwiftLintFramework/Rules/ForceCastRule.swift]: right, but this is a non-triggering example?

Comment (jpsim) [inline Source/SwiftLintFramework/Rules/ForceCastRule.swift]: `verifyRule()` will treat all triggering examples as non-triggering when commented out. So `// NSNumber() as! Int` was already being tested because `NSNumber() as! Int` is a triggering example. So this line was duplicating a test.
